test/agentchat/contrib/vectordb/test_mongodb.py

```diff
@@ -0,0 +1,288 @@
+import logging
+import os
+from time import sleep
+from typing import List
+
+import pytest
+
+from autogen.agentchat.contrib.vectordb.base import Document
+
+try:
```

Comment:
The test is not shown in the RetrieveChatTest.
![image](https://github.com/microsoft/autogen/assets/3197038/a836c7bd-a19b-4c67-8fc3-e0ee5ba4c63c)
https://github.com/microsoft/autogen/actions/runs/9732248031/job/26857669616?pr=3000